cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -30,6 +30,36 @@ Cask Data Application Platform Release Notes
    :backlinks: none
    :depth: 2
 
+`Release 5.1.1 <http://docs.cask.co/cdap/5.1.1/index.html>`__
+=============================================================
+
+Improvements
+------------
+
+- :cask-issue:`CDAP-14490` - Google Cloud Spanner sink will create database and table if they do not exist.
+
+- :cask-issue:`CDAP-14542` - Added a Dataset Project config property to the Google BigQuery source to allow reading from a dataset in another project.
+
+
+Bug Fixes
+---------
+
+- :cask-issue:`CDAP-10250` - Fixed configuration isolation in mapreduce.
+
+- :cask-issue:`CDAP-12229` - Fixed an issue that caused avro, parquet, and orc classes across file, Google Cloud Storage, and S3 plugins to clash and cause pipeline failures.
+
+- :cask-issue:`CDAP-14511` - Fixed a bug where plugins that register other plugins would not use the correct id when using the PluginSelector API.
```

Comment:
ditto

Reply:
I don't think we do line break for the previous version release notes. Can you confirm we want to do that?